core/templates/dev/head/components/response_header_directive.html

```diff
@@ -45,8 +45,8 @@
 </style>
 <div class="oppia-response-header-block">
   <div class="oppia-response-header">
-    <i class="material-icons md-18" ng-if='!getCorrect() && isCorrectnessFeedbackEnabled()'>&#x2718;</i>
-    <i class="material-icons md-18" ng-if='getCorrect() && isCorrectnessFeedbackEnabled()'>&#10004;</i>
+    <i class="material-icons md-18" ng-if='!getCorrect() && isCorrectnessFeedbackEnabled() && isResponse()'>&#x2718;</i>
```

Comment:
Just a small nit, can you change the single quotes into double quotes here to be consistent with the other attr values? Ditto for the line below. I know this existed before but we might as well fix it.

Reply:
Done!